docs/api.rst

```diff
@@ -0,0 +1,8 @@
+General API customization
+=========================
+
+.. confval:: include_object_description_fields_in_toc
+
+   :python:`bool` indicating whether to include domain object description
+   fields, like "Parameters", "Returns", "Raises", etc. in the table of
+   contents.  Defaults to :python:`True`.
```

Comment:
Different people learn differently. While I'm ok with reading this paragraph and understanding this option's function, some might benefit from a simple example.

Instead of demonstrating all forms of the fields effected, a `seealso` link (or list/table of links) to a demonstration of this would be a good idea. However, due to #58 , it might be better to have a localized example, so the right toc isn't confusing when following a link.

Reply:
I added some clarification and a link to an example.